components/Button.tsx

```diff
@@ -7,3 +10,31 @@ export const Button = React.forwardRef<
   return <button {...props}>{children}</button>;
 });
 Button.displayName = 'Button';
+
+interface SmallRoundedButtonProps {
+  $primaryColor?: string;
+  $secondaryColor: string;
+}
+
+export const SmallRoundedButton = styled.button<SmallRoundedButtonProps>`
```

Comment:
maybe we can add a comment explaining what $primaryColor and $secondaryColor do!